src/modules/map/components/openlayers/OpenLayersVectorLayer.vue

```diff
@@ -34,10 +37,18 @@ export default {
             default: null,
         },
     },
+    computed: {
+        mapLibreInstance() {
+            return this.layer?.maplibreMap
+        },
```

Comment:
Also from past experience we should whenever possible avoid using complex object in the vue reactive system, I don't know about this `maplibreMap` object but it might be quite complex or could become complex in future depedencies updates.